src/Zinc-HTTP/ZnEasy.class.st

```diff
@@ -86,7 +86,8 @@ ZnEasy class >> getImageOfType: mimeType fromUrl: urlObject [
 		enforceAcceptContentType: true;
 		get.
 	"ImageReadWriter does automatic type detection"
-	^ ImageReadWriter formFromStream: client entity readStream
+	"ImageReadWriter is not in minimal image"
+	^ (self environment at: #ImageReadWriter) formFromStream: client entity readStream
```

Comment:
See #4000.

Reply:
Hi, probably @svenvc would like to see this change.

I think a better solution would be to put this method into a separate package? something like Zinc-HTTP-Images?

Reply:
Feel free to submit a PR doing something else.